content/hardware/recommended/camera-gimbal/index.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Camera Tilt Mount / Gimbal"
+description = "Used to move and/or stabilise the camera, for smoother video feeds and looking around without needing to rotate or move the whole vehicle."
+date = 2022-10-11T17:33:19+11:00
+template = "docs/page.html"
+sort_by = "weight"
+weight = 10
+draft = false
+[extra]
+lead = ""
+toc = true
+top = false
++++
+
+
+ArduSub supports control and attitude atabilization for up to 3-axis gimbals. The most common gimbal type for underwater vehicles is a 1-axis tilt mount, since the majority of standard ROV frames have control over the other Degrees of Freedom (DoF).
+
+ArduSub supports both simple servo-driven (PWM) gimbals in which the autopilot's onboard IMU controls the stabilisation and brushless direct drive (UART) gimbals that have their own self-stabilization controllers.
```

Comment:
maybe mention mavlink gimbals too. like the storm32

Reply:
That's already covered by the "brushless UART gimbals that have their own self-stabilization controllers", and is already linked to as one of the supported options. I'll add an explicit note that that one uses MAVLink :-)